cfpb/consumerfinance.gov
# Buying a House: Migrate deprecated keyCode to KeyboardEvent.key

## Changes

- Buying a House: Migrate deprecated keyCode to KeyboardEvent.key.


## How to test this PR

1. Check http://localhost:8000/owning-a-home/explore-rates/ and see that loan amount isn't updated when the enter, arrow, shift, or tab keys are pressed. A non-numeric value can't be entered.
